src/components/ConfirmationModal/_confirmation-modal.scss

```diff
@@ -48,7 +48,7 @@
   }
 
   &__info-group {
-    margin-bottom: $spv-outer--medium;
+    margin-bottom: $spv--large;
```

Comment:
`$spv-outer--*` are removed, the value of `$spv-outer--medium` matches the value of `$spv--large`